django/db/models/utils.py

```diff
@@ -19,3 +22,16 @@ def make_model_tuple(model):
             "Invalid model reference '%s'. String model references "
             "must be of the form 'app_label.ModelName'." % model
         )
+
+
+def get_field_from_path(model, path):
```

Comment:
This could do with a docstring, and tests I think. 

What happens if `path` contains an error? e.g. `"user__groups__not_there"` Do we need to handle that at all?

Reply:
Unless this function can be used elsewhere (In which case I would introduce it and use it in a separate PR), I would leave this as a private helper function in `views/generic/utils.py`